src/open_inwoner/js/components/rich-text-editors/ckeditor-hide-notification.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * TODO: Remove this when the project switches to a different rich text editor.
+ * Class to handle CKEditor warning notification on the note-edit page for plans.
+ */
+export class CkeditorNotificationHider {
+  /**
+   * @param {HTMLElement} context Optional root context (defaults to document.body).
+   */
+  constructor(context = document.body) {
+    this.context = context
+    this.init()
+  }
+
+  /**
+   * Start observing and hiding the warning.
+   */
+  init() {
+    this.observer = new MutationObserver(() => this.hideWarning())
+
+    this.observer.observe(this.context, {
+      childList: true,
+      subtree: true,
+    })
+
+    // Just in case it's already rendered by the time this runs.
+    this.hideWarning()
+  }
+
+  /**
+   * Hides the CKEditor version warning if it exists.
+   */
+  hideWarning() {
+    const warning = this.context.querySelector(
+      '.view--collaborate-plan_edit_note .cke_notification_warning'
```

Comment:
@stefrado This `.view--collaborate-plan_edit_note class` on the HTML element makes sure this script is only executed on this 1 unique page in the project.